src/main/java/com/laba/dal/mybatisDAOs/AppointmentMbDAO.java

```diff
@@ -0,0 +1,15 @@
+package com.laba.dal.mybatisDAOs;
```

Comment:
```suggestion
package com.laba.dal.mybatisdaos;
```
package name - all lowercase